appengine-java11/appengine-simple-jetty-main/src/main/java/com/example/appengine/demo/jettymain/Main.java

```diff
@@ -36,7 +36,11 @@ public static void main(String[] args) throws Exception {
     // Note: If you set this to port 0, a randomly available port will be
     // assigned. You can find the assigned port in the logs or programmatically
     // obtain it.
-    Server server = new Server(8080);
+    Server server = new Server(
```

Comment:
This doesn't look like it's handling any exceptions thrown. 

I'd also prefer to see something like the following:
```java
int port = Integer.parseInt(System.getenv().getOrDefault("PORT", "8080"));
Server server = new Server(port);
```

Multi-line ternary statements quickly become hard to read.

Reply:
Changes done!
and improve the code comments.